rpcd/playbooks/maas-20151013-2-pre-migrate.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Part of MaaS Migration: maas-20151013.yml
+- name: Record Prior Checks before MaaS Migration
+  hosts: hosts
+  user: root
+  tasks:
+    - name: Inspect existing checks - all entities
+      script: "{{ maas_rpc_scripts_dir }}/rpc-maas-tool.py --tab True checks"
```

Comment:
The var "maas_rpc_scripts_dir" is used outside of the rpc_maas role so its not actually present. We'd need to add this to user_extras_variables.yml or group_vars, or some other place where it is accessible as a var outside of the role.